docs/20_samples/10_sample-edition/molecule/README.md

```diff
@@ -0,0 +1,56 @@
+# Molecule
+
+:::info Overview
+
+![overview](overview.png)
+
+1. If you write a title in this tab, it will be automatically updated in the `Title` tab of the over views and will appear in the final report. [Click here](../includes/titletab/README.md) for more information.
+2. The keywords appear in the main table of the homepage. You would be able to search for your sample in the homepage according to those keywords.
+3. You can change the structure of your sample here. The new molecular formula and molecular weight would be automatically calculated and the structure would be updated in all the view of this tile. 
+4. If you change the molecular formula, the new one would appear in the report.
+5. The canonized molecular formula depends directly on the molecular formula. It is updated every time the molecular formula changes. 
+6. Electromagnetic mass and molecular weight, calculated from the molecular formula.
```

Comment:
>Electromagnetic mass

is this correct?

Reply:
I am not sure. @lpatiny could you confirm ?

Reply:
i think it is supposed to be the exact mass

Reply:
https://en.wikipedia.org/wiki/Mass_(mass_spectrometry)#Exact_mass